openforcefield/tests/test_topology.py

```diff
@@ -605,6 +606,11 @@ def test_from_openmm(self):
             get_data_file_path("systems/packmol_boxes/cyclohexane_ethanol_0.4_0.6.pdb")
         )
 
+        with pytest.raises(
+            MissingUniqueMoleculesError, match="requires a list of Molecule objects"
+        ):
+            Topology.from_openmm(pdbfile.topology, unique_molecules=None)
```

Comment:
Could we remove the `unique_molecule=None` part? What we mean to test is that the _default_ argument alone is unacceptable, and some day we might make the default an empty list, but still want to fail if it's empty.